src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -567,4 +567,92 @@
   <data name="SwitchSelectionEndpointCommandKey" xml:space="preserve">
     <value>Switch selection endpoint</value>
   </data>
+  <data name="ColorSelection_allMatches" xml:space="preserve">
+    <value>, all matches</value>
+    <comment>This is used in the description of an action which changes the color of selected text, to indicate that not only the selected text will be colored, but also all matching text.</comment>
+  </data>
+  <data name="ColorSelection_fg_action" xml:space="preserve">
+    <value>Color selection, foreground: {0}{1}</value>
+    <comment>This is the description of an action which changes the color of selected text. {0}: the color. {1}: empty string OR a clause indicating that all matching text will be colored (ColorSelection_allMatches).</comment>
+  </data>
+  <data name="ColorSelection_bg_action" xml:space="preserve">
+    <value>Color selection, background: {0}{1}</value>
+    <comment>This is the description of an action which changes the background color of selected text. {0}: the color. {1}: empty string OR a clause indicating that all matching text will be colored (ColorSelection_allMatches).</comment>
+  </data>
+  <data name="ColorSelection_fg_bg_action" xml:space="preserve">
+    <value>Color selection, foreground: {0}, background: {1}{2}</value>
+    <comment>This is the description of an action which changes the color of selected text and the background. {0}: the foreground color. {1}: the background color. {2}: empty string OR a clause indicating that all matching text will be colored (ColorSelection_allMatches).</comment>
+  </data>
+  <data name="ColorSelection_default_action" xml:space="preserve">
+    <value>Color selection, (default foreground/background){0}</value>
+    <comment>This is the description of an action which changes the color of selected text and the background to the default colors. {0}: empty string OR a clause indicating that all matching text will be colored (ColorSelection_allMatches).</comment>
+  </data>
+  <data name="ColorSelection_defaultColor" xml:space="preserve">
+    <value>[default]</value>

```

Comment:
nit: let's move the `[` and `]` into the code instead of the localized resource

Reply:
Ok, I can't figure out how to get the `[default]` to pop up in the command palette. It seems like whenever `ColorSelectionArgs::Foreground()` or `ColorSelectionArgs::Background()` are `nullptr`, we end up adding something like `Color selection, (default foreground/background)` instead.

So I'm leaving it like this. If you have an example JSON to get that `[default]` to show up, please post it here. Otherwise, if it's impossible to present it, we should probably remove it.